src/setupTests.ts

```diff
@@ -4,6 +4,15 @@ import { cleanup } from '@testing-library/react';
 import nock from 'nock';
 import { vi, afterEach, expect, afterAll } from 'vitest';
 
+vi.mock('./server/helpers/env.ts', async (importOriginal) => {
+  const envModule = (await importOriginal()) as any;
+  return {
+    ...envModule,
+    // Prevent isRequired from spamming logs or throwing errors by ignoring it.
+    getFromEnv: (key: string, _isRequired: boolean = true) => process.env[key],
```

Comment:
is het nodig isREquired te typen? lijkt me dat die geinferred wordt

Reply:
HUH, dat kan haha, ik verwacht altijd een error als iets gecalled word met teveel argumenten.
![afbeelding](https://github.com/user-attachments/assets/62130155-3ff4-433b-9cde-b56cece58fbb)
waaaaat :D

Het is verwijderd, bedankt voor de review